### DISCUSSION TITLE
HISPlayer Network error

### DISCUSSION THREAD

**Discussion by Ramadhan2020** [HISPlayer/Unity_Video_Player#9] [Q&A]
I try to create a build using HISPlayer Unity SDK  the player working find on Unity Editor but when I build to android I got this error message on the logcat and video fail to run

`2024/07/15 21:58:43.753 23889 23915 Error Unity HISPLAYER - ERROR: Network error, impossible to initialize HISPlayer SDK. Please, check your wifi or mobile data
`

I'm pretty sure my network and mobile data is working without problem during the test, My Android version is Android 13  (API level 33)

The asset that I'm using is from this link
https://assetstore.unity.com/packages/tools/video/video-streaming-player-for-short-videos-262766#reviews

Can anyone help me with this?

**Comment by NurikHISPlayer** [comment]
Hi @Ramadhan2020, 
That error actually comes when you don't have internet connection during the player initialization and the license server can't be reached, can you retry it, please?  Let us know the result.

**Reply by Ramadhan2020** [reply]
I try it several times, the same error still happen. I also try to stream youtube using my tester android phone and youtube video running without problem so my internet is ok.

But the SDK does not want to run and the same error message keep showing.

**Reply by NurikHISPlayer** [reply]
What HISPlayer SDK version are you using? If it is 3.1.1, please move to 3.4.1 (the one in your asset store url above), that is a known issue at 3.1.1. Thanks.

**Reply by Ramadhan2020** [reply]
this is the SDK I'm using
![image](https://github.com/user-attachments/assets/d590e8c5-c389-4634-8a54-a447cb620ea0)

Test Device is smartphone model Infinix GT 10 Pro

**Reply by NurikHISPlayer** [reply]
Did you move to 3.4.1. from an older version?

**Reply by NurikHISPlayer** [reply]
the log you sent doesnt exist in version 3.4.1, it seems to be related to an older version. Please in that case make sure to update the SDK properly by removing the old package. Please refer to: https://hisplayer.github.io/UnityAndroid-SDK/#/update-the-sdk (you may ignore the license key update part). Let us know how that helps.
